### ISSUE TITLE
Cannot download unofficial modpacks

### ISSUE THREAD

**Issue by AbstractRude** [wabbajack-tools/wabbajack#2142]
**Describe the bug**

Cannot download modpacks - I click the download button, I see web requests in the upper right hand corner of the screen for a few seconds, but then nothing. The modpack never downloads. I can download MOST official modpacks fine, but no unofficial ones. 

**Additional Context**

Windows 11 - Skyrim Anniversary Edition

**Wabbajack Version**

3.0.3.0

**Logs**

[Wabbajack.current.log](https://github.com/wabbajack-tools/wabbajack/files/9874137/Wabbajack.current.log)

**Comment by EzioTheDeadPoet** [comment]
Do you have a list of which unofficial packs you tried ?

Or would all of them be listed in this log ?

**Comment by AbstractRude** [comment]
> Do you have a list of which unofficial packs you tried ?
> 
> Or would all of them be listed in this log ?

Lost Legacy
Masterstoke (Not unofficial, I believe? But one that just doesn't work, but it worked before the 25th)
Nefaram
Happy Fun Times
Skybax
Do Not Go Gentle
Elysium Remastered

I stopped trying around that point. Just now, I decided to try a few more unnofficial ones, and some of them do work fine, so I suppose the name of my thread is a bit wrong here.
